docs/applications/nublado/major-upgrade.rst

```diff
@@ -0,0 +1,313 @@
+""""""""""""""""""""
+
+Remove any user sessions.  This is not, strictly speaking, necessary,
+but if you have a user with a running v2 installation who decides to use
+the v3 endpoint, they will risk the file corruption mentioned above.
+You can do this by deleting the namespace objects in the
+``nublado-users`` application (which will also clean up anything within
+those namespaces) or by executing something like ``kubectl get ns | grep
+nublado2- | awk '{print $1}' | xargs kubectl delete ns``.
+
+Bring up applications
+"""""""""""""""""""""
+#. Synchronize the ``science-platform`` application.  This will enable
+   the ``nublado`` application, which contains nublado v3.
+#. Synchronize the ``nublado`` application.  This will turn on the
+   nublado v3 spawner.
+#. Test that you can get a lab and that that lab allows use of the
+   command line and cell executions with the new spawner at ``/n3``.  If
+   you find errors, troubleshoot them.  Only proceed once you have the
+   new spawner correctly spawning and shutting down labs.
+#. Synchronize the ``nublado2`` application.  This will reenable the
+   default spawner.  At this point users will start returning (whether
+   or not you call an all-clear, which you should not yet; users don't
+   pay attention to such things).
+#. Re-enable mobu, if you disabled it.  Wait to make sure it's correctly
+   running notebooks, or test a few yourself.
+
+Make nublado v3 the default
+---------------------------
+
+Pre-deployment Preparation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+As before, these steps can be done at any time, and should be done in
+advance of the maintenance window.
+
+Communicate to user base
+""""""""""""""""""""""""
+
+Let your users know that ``/nb`` will refer to the new spawner, and that
+the old one will be available for a while at ``/n2``, as well as the
+timeframe for these changes (presumably, at the end of the next
+maintenance window).  Announce that that maintenance window will be
+disruptive and will destroy any existing user sessions.
+
+If you are using the same storage for users regardless of which spawner
+they're coming in as, which you very likely are, in addition to the new
+route, you need to do your utmost to make it extremely clear that if
+they intend to use both routes, it is vital to make sure that their labs
+are shut down before changing from one route to the other.  If they do
+not do this, it will at best create confusion as open notebooks will be
+saved every five minutes by each environment, thus bafflingly reverting
+changes, and at worst it will cause full-scale corruption of the files
+the user has open.
+
+Change mobu configuration
+"""""""""""""""""""""""""
+
+If you are using ``mobu``, change the instance-specific values file.  In
+``config.autostart``, for each entry, set ``business.use_cachemachine`` to
+``false``.  This is necessary because nublado 3 uses its Jupyterlab
+Controller to carry out the functions formerly performed by
+``cachemachine``.
```

Comment:
and turn mobu back on and sync it, or however you're turning it on (I think you're implying that here)

Reply:
I'll make it explicit that it's a resync.